site/docs/latest/cookbooks/RetentionExpiry.md

```diff
@@ -97,7 +97,7 @@ $ pulsar-admin namespaces set-retention my-prop/my-cluster/my-ns \
 
 #### REST API
 
-{% endpoint POST /admin/namespaces/:property/:cluster/:namespace/retention %}
+{% endpoint POST /admin/v2/namespaces/:property/:cluster/:namespace/retention %}
```

Comment:
In all v2 APIs, the portion `:property/:cluster/:namespace` would actually be `:property/:namespace`, since we actually took the cluster out of the namespaces (and topic) names.

Reply:
And the terminology was changed from `property` into `tenant`

Reply:
Done